sky/packages/sky/lib/src/rendering/stack.dart

```diff
@@ -226,14 +230,112 @@ class RenderStack extends RenderBox with ContainerRenderObjectMixin<RenderBox, S
     defaultHitTestChildren(result, position: position);
   }
 
+  void paintStack(PaintingContext context, Offset offset);
+
   void paint(PaintingContext context, Offset offset) {
     if (_hasVisualOverflow) {
       context.canvas.save();
```

Comment:
Nope, but we should create a bug and comment each case we see with a reference to the bug.